controllers/clustertopology_controller_compute_test.go

```diff
@@ -250,7 +252,281 @@ func TestGetClass(t *testing.T) {
 	}
 }
 
-func TestGetTemplate(t *testing.T) {
+func TestGetCurrentState(t *testing.T) {
+	fakeScheme := runtime.NewScheme()
```

Comment:
This block is very hard to read. I think a few spaces to structure it would be really nice

Reply:
I'll add spaces and some comments to summarize what's being created in the fake.... block